server/AppointmentPlanner/Shared/MainLayout.razor

```diff
@@ -161,6 +161,6 @@
 
     private void showSideBar()
     {
-        open = !open;
+        open = true;
```

Comment:
Breaking reason?

Reply:
https://github.com/essential-studio/blazor-showcase-appointment-planner/blob/2616911a1aa2e5e3a04e91bfc061ca53d7ebf033/server/AppointmentPlanner/Shared/MainLayout.razor#L164 - I don't know the reason of this change, I have just moved the changes from latest changes having release/26.1.1 repo to public repo.